vcpkg-example/main.cpp

```diff
@@ -0,0 +1,14 @@
+﻿#include <iostream>
+#include <optional>
```

Comment:
These should not be needed and can be removed.

Reply:
Do you mean we don't need `main.cpp` or just part of the code? Or just a simple `main.cpp`?

Reply:
Just the two lines I highlighted ;)